src/Engine/ProtoCore/SyntaxAnalysis/AstVisitor.cs

```diff
@@ -477,6 +477,19 @@ public override AssociativeNode VisitFunctionCallNode(FunctionCallNode node)
             return node;
         }
 
+        public override AssociativeNode VisitLanguageBlockNode(LanguageBlockNode node)
```

Comment:
This is adding support for `AST` rewrite for namespace resolution of classes used inside Associative language blocks. Earlier these expressions used inside explicit `[Associative]` language block were ignored.